O/Openresty/build_tarballs.jl

```diff
@@ -3,14 +3,14 @@
 using BinaryBuilder, Pkg
 
 name = "Openresty"
-version = v"1.19.9"
+version = v"1.21.4"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://openresty.org/download/openresty-1.19.9.1.tar.gz", "576ff4e546e3301ce474deef9345522b7ef3a9d172600c62057f182f3a68c1f6"),
-    ArchiveSource("https://ftp.pcre.org/pub/pcre/pcre-8.43.tar.gz", "0b8e7465dc5e98c757cc3650a20a7843ee4c3edf50aaf60bb33fd879690d2c73"),
-    ArchiveSource("https://www.openssl.org/source/openssl-1.1.1l.tar.gz", "0b7a3e5e59c34827fe0c3a74b7ec8baef302b98fa80088d7f9153aa16fa76bd1"),
-    ArchiveSource("https://www.zlib.net/zlib-1.2.11.tar.gz", "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1")
+    ArchiveSource("https://openresty.org/download/openresty-$(version).1.tar.gz", "0c5093b64f7821e85065c99e5d4e6cc31820cfd7f37b9a0dec84209d87a2af99"),
+    ArchiveSource("https://sourceforge.net/projects/pcre/files/pcre/8.45/pcre-8.45.tar.bz2", "4e6ce03e0336e8b4a3d6c2b70b1c5e18590a5673a98186da90d4f33c23defc09"),
```

Comment:
Why are we not using these as proper dependencies instead?

Reply:
You mean like using jlls?

Reply:
Sources are needed because these are statically compiled into openresty binary